.teamcity/_self/projects/WebApp.kt

```diff
@@ -123,6 +131,10 @@ object BuildDockerImage : BuildType({
 					--build-arg node_memory=32768
 					--build-arg use_cache=true
 					--build-arg base_image=%base_image%
+					--build-arg commit_sha=${Settings.WpCalypso.paramRefs.buildVcsNumber}
```

Comment:
At which point is the commit hash assigned to this var? The name seems to suggest a (possibly sequential?) number, maybe it's part of the regular TC API and was overridden by us to contain the commit hash? Just curious.

Reply:
It seems to be used elsewhere in the file to reference the commit sha. I think it could mean "the VCS identifier used in the current build," which would be the current SHA in git. I don't think we're overriding it anywhere, as far as I can tell, and it does inject the correct value into the build.